### PR TITLE
Leap year code in dart

### PR DESCRIPTION
**Pull request by ghost**
Added a Dart program to check leap years and a README file with instructions for #18

### CODE REVIEW

**Comment by andrewhoyer** [comment]
Can you make one addition to your PR? Please a link to your Dart folder in the main README file of the repo. That way there's a link to it on the main page below the list of languages.

This is noted in the CONTRIBUTING file, but it's not totally clear, so I'm going to fix that part as well.

Please update the PR and I will review it again.

File: Dart/README.md
Comment (andrewhoyer) [inline Dart/README.md:1]: On line 10, there's the start of a code block, but there is no end to it, so it includes the rest of the text. 

Three backticks should follow the dart command, based on the code which do not produce the following text as output.

**Review by andrewhoyer — CHANGES_REQUESTED** [review]
Generally looks good. Just one comment about the README formatting.

**Comment by ghost** [comment]
Thank you for assisting. Please let me know if any other changes are required.

**Review by andrewhoyer — APPROVED** [review]

**Comment by andrewhoyer** [comment]
Thank you for your contribution!

**Pull request merged**